CMakeLists.txt

```diff
@@ -259,6 +259,10 @@ install ( TARGETS ${nlopt_lib}
           ARCHIVE DESTINATION ${RELATIVE_INSTALL_LIB_DIR}
         )
 
+if (MSVC AND BUILD_SHARED_LIBS AND ${CMAKE_MAJOR_VERSION} GREATER 2 AND ${CMAKE_MINOR_VERSION} GREATER 0)

```

Comment:
rather do ```CMAKE_VERSION VERSION_GREATER 3.1```

Reply:
The minimum version is 3.1, I think `NOT CMAKE_VERSION VERSION_LESS 3.1` would be better in this case?

Reply:
Indeed, VERSION_LESS is the strict comparison